src/components/Navbar.js

```diff
@@ -1,18 +1,23 @@
 import React from "react";
 import { Link } from "react-router-dom";
+import { Button, TextField, Paper } from "@mui/material";
  
 
 
 const Navbar = () => {
   return (
-        <nav className="navbar">
-            .
-            <Link to ='/'>Home</Link>
-            <Link to ='/worlds'>Campaign List</Link>
-            <Link to ='/createnewcampaign'>Create New Campaign</Link>
-            <Link to ='/about'>About</Link>
-            .
-        </nav>
+        <Paper sx={{
+            height: "40px"
```

Comment:
Hey @IsaacScheff,
In most situations, you'd want to use `em` or `rem` units instead of pixels. They result in a more consistent user experience across different devices.

Reply:
Also, since you're setting height for the container of the nav bar, you probably want to give it a min-height so that you won't have to change it every time you add or remove an item.